tests/periph_gpio/main.c

```diff
@@ -55,11 +57,48 @@ static int init_pin(int argc, char **argv, gpio_mode_t mode)
     return 0;
 }
 
+static int init_pin_hl(int argc, char **argv, bool high)
+{
+    int po, pi, r;
```

Comment:
```suggestion
    int port, pin, r;
```

Only three letters more, but much more readable ;-)

Reply:
I also changed `init_pin` while I'm at it.

Reply:
No. I somehow assumed this to be `static inline` in the header. I was scrolling to fast over the diff :-/